src/msw/textctrl.cpp

```diff
@@ -1114,6 +1114,115 @@ void wxTextCtrl::DoSetValue(const wxString& value, int flags)
     }
 }
 
+#if wxUSE_RICHEDIT
+
+DWORD wxCALLBACK MSWEditStreamOutCallback(DWORD_PTR dwCookie, LPBYTE pbBuff,
+    LONG cb, LONG* WXUNUSED(pcb))
+{
+#if wxUSE_THREADS
+    wxMutexGuiLeaveOrEnter();
+#endif // wxUSE_THREADS
+    MSG msg;
+    while ( ::PeekMessage(&msg, (HWND)0, 0, 0, PM_REMOVE) )
+    {
+        if (msg.message != WM_QUIT)
+        {
+            ::TranslateMessage(&msg);
+            ::DispatchMessage(&msg);
+        }
+    }
+
+    // write the text
+    std::string* psEntry = reinterpret_cast<std::string*>(dwCookie);
+    if ( psEntry != nullptr )
```

Comment:
Can it actually be null here? AFAICS it can't and then this test should be either replaced with a `wxCHECK` or removed entirely.

Reply:
No, it couldn't. Just being overly pedantic. I'll remove that check.